src/account/SecuritySubmenu.test.tsx

```diff
@@ -0,0 +1,347 @@
+import * as React from 'react'
```

Comment:
This whole file is essentially verbatim copied over from `Settings.test.tsx`, save for some small superficial changes.